baselines/nadamw/tuning_search_space.json

```diff
@@ -1,37 +1,20 @@
 {
-    "learning_rate": {
-        "feasible_points": [
-            0
-        ]
-    },
-    "beta1": {
-        "feasible_points": [
-            0
-        ]
-    },
-    "beta2": {
-        "feasible_points": [
-            0
-        ]
-    },
-    "warmup_steps": {
-        "feasible_points": [
-            0
-        ]
-    },
-    "weight_decay": {
-        "feasible_points": [
-            0
-        ]
-    },
-    "label_smoothing": {
-        "feasible_points": [
-            0
-        ]
-    },
-    "grad_clip": {
-        "feasible_points": [
-            0
-        ]
-    }
+  "learning_rate": {
```

Comment:
add beta2 : 0.999 
add epsilon : 1e-8

Reply:
Same as above, no need for epsilon. But good catch on beta2